src/adapters/ui/iconview.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable linebreak-style */
```

Comment:
Why was this eslint disable added btw ?

Reply:
Oh crud, sorry. New to working in vscode and with the fancy tools, currently working on Windows and ESLint threw a fit over line breaks, i'll remove and recommit.

Reply:
.. I'll setup a devcontainer later too so I can just avoid those headaches.